spec/linter-slim-spec.js

```diff
@@ -0,0 +1,51 @@
+'use babel';
+
+import * as path from 'path';
+import {
+  // eslint-disable-next-line no-unused-vars
+  it, fit, wait, beforeEach, afterEach,
+} from 'jasmine-fix';
+
+const { lint } = require('../lib/linter-slim.js').provideLinter();
+
+const goodFile = path.join(__dirname, 'fixtures', 'good.slim');
+const badFile = path.join(__dirname, 'fixtures', 'syntax_error.slim');
+const deprecatedFile = path.join(__dirname, 'fixtures', 'deprecation_warning.slim');
+
+describe('The slimrb provider for Linter', () => {
+  beforeEach(async () => {
+    atom.workspace.destroyActivePaneItem();
+    await atom.packages.activatePackage('language-slim');
+    await atom.packages.activatePackage('linter-slim');
+  });
+
+  it('checks a file with syntax error and reports the correct message', async () => {
+    const excerpt = 'SyntaxError: Unknown line indicator';
+    const editor = await atom.workspace.open(badFile);
+    const messages = await lint(editor);
+
+    expect(messages.length).toBe(1);
+    expect(messages[0].severity).toEqual('error');
+    expect(messages[0].excerpt).toEqual(excerpt);
+    expect(messages[0].location.file).toBe(badFile);
+    expect(messages[0].location.position).toEqual([[4, 2], [4, 9]]);
+  });
+
+  it('fchecks a file with a deprecation warning and reports the correct message', async () => {
```

Comment:
I think this should be as follows, right?
```suggestion
  it('checks a file with a deprecation warning and reports the correct message', async () => {
```